packages/react-reconciler/src/ReactFiberWorkLoop.old.js

```diff
@@ -401,7 +401,19 @@ export function requestUpdateLane(fiber: Fiber): Lane {
     return pickArbitraryLane(workInProgressRootRenderLanes);
   }
 
-  const isTransition = requestCurrentTransition() !== NoTransition;
+  const eventPriority: EventPriority = (getCurrentUpdatePriority(): any);
+
+  // Check if the event priority is a lane, by checking if it's a power of 2. If
+  // it's not, then we assume it's TransitionPriority. This is a trick to avoid
+  // leaking the Lane type into the shared reconciler package. Consumers should
+  // be versioning the isomorphic package in lockstep with the renderer — but
+  // for those rare exceptions where there's a mismatch (like, say, a
+  // multi-renderer app), this is less likely to break than if we used an
+  // arbitrary lane bit that later changes meaning.
+  // TODO: Move this type conversion to the event priority module.
+  const maybePowerOfTwo = (eventPriority: any);
+  const isTransition =
+    maybePowerOfTwo !== 0 && (maybePowerOfTwo & (maybePowerOfTwo - 1)) !== 0;
```

Comment:
Idk, this seems overly clever Andrew. Maybe should just compare to magic number 3. It won't change.